source/common/html/utility.cc

```diff
@@ -0,0 +1,37 @@
+#include "common/html/utility.h"
+
+#include <string>
+
+namespace Envoy {
+namespace Html {
+
+/* static */
```

Comment:
nit: This comment is not something we typically do in the codebase.

Reply:
done